tests/rust_protobuf/v2/test_enum_values_pb.rs

```diff
@@ -3,6 +3,9 @@
 #![allow(non_snake_case)]
 #![allow(non_upper_case_globals)]
 #![allow(non_camel_case_types)]
+#![allow(unknown_lints)]
+#![allow(clippy)]
+#[cfg_attr(rustfmt, rustfmt_skip)]
```

Comment:
Why not `#![cfg_attr(rustfmt, rustfmt_skip)]`? It applies to the item enclosing it, in this case the item is `mod test_enum_values_pb`. Also I guess it will fix `error: expected item after attributes`.

See more: https://doc.rust-lang.org/book/attributes.html